eventdiscovery-sdk/src/main/res/values/colors_light.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Colors of the light theme -->
+
+    <color name="schedjoules_colorPrimary">@color/schedjoules_white</color>
```

Comment:
Shouldn't we then get rid of all these 'intermediaries' that are used as theme attributes? Because if theme attributes are overriden by partner, then these 'detached' references are still used using the old color. Just as with the details icons. So in order to avoid that, shouldn't we remove these and instead reference the theme attribute wherever these are used?
(This what I wanted to say in the previous post, where I confusingly said 'hardcoding' the theme colors.)
This could of course be a separate task.